index.html

```diff
@@ -151,15 +151,15 @@ <h2 class="card-title">Multi-Post Stories</h2>
         </div>
 </section>
 
-<section id="work-section-desktop">
+<section id="work-section-desktop tonic">
     <div class="card">
-        <div class="card-img">
-            <img src="./images/Snapshoot Portfolio.png" alt="tonic image">
+        <div class="card-img tonic">
+            <img src="./images/desktop-img/Snapshoot Portfolio(1).png" alt="tonic image">
         </div>
         <div class="card-description">
             <h2 class="card-title">tonic</h2>
             <ul class="canopy">
-                <li>canopy</li>
+                <li id="canopy-text">canopy</li>
                 <li>Back End Dev</li>
                 <li>2015</li>
             </ul>
```

Comment:
Congratulations on doing an excellent job incorporating the feedback from the previous reviewer! **I have one additional suggestion that you might find helpful if you're interested in making your work cards more closely resemble the[ Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=ylRy5lsCCVeUSa89-0).** This suggestion is completely optional, but **I believe it could further enhance the polished and professional look of your work.**

- [ ] **[OPTIONAL]** It appears that there may be an issue with the spacing between the `title` of your card and the card `details` at the bottom. Can you please double-check and ensure that there **is an appropriate amount of space between these two elements** to ensure readability and a polished appearance? 😉

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236528230-ca0a62a5-2c54-435c-98ad-72bf065cb7d2.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236528407-ab4b4e38-4133-41f6-8849-275d76917834.png)